_includes/page__hero_video.html

```diff
@@ -0,0 +1,17 @@
+{% include base_path %}
+
+{% capture video_id %}{{ page.header.video.id }}{% endcapture %}
+{% capture video_provider %}{{ page.header.video.provider }}{% endcapture %}
+
+<div class="page__hero{% if page.header.overlay_color or page.header.overlay_video_path %}--overlay{% endif %}"
+  style="{% if page.header.overlay_color %}background-color: {{ page.header.overlay_color | default: 'transparent' }};{% endif %} {% if overlay_video_path %}background-video_path: {% if overlay_filter %}linear-gradient({{ overlay_filter }}, {{ overlay_filter }}), {% endif %}url('{{ overlay_video_path }}');{% endif %}"
+>
+
+{% if video_provider == "vimeo" %}
+    <style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='http://player.vimeo.com/video/{{ video_id }}' frameborder='0' webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe></div>
```

Comment:
This is invalid Liquid for "else if". Should be `{% elsif video_provider == "youtube" %}` instead.